From e55c8c22551f4410c6d600df9f133ff13cfa20cf Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Tue, 19 May 2020 17:21:42 -0600
Subject: [PATCH 2/6] Add xxxhdpi icon_webview.png to BUILD.gn

---
 android_webview/nonembedded/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/android_webview/nonembedded/BUILD.gn b/android_webview/nonembedded/BUILD.gn
index 02bf56f150614..4f58b36a2e758 100644
--- a/android_webview/nonembedded/BUILD.gn
+++ b/android_webview/nonembedded/BUILD.gn
@@ -179,6 +179,7 @@ android_resources("icon_resources") {
     "java/res_icon/drawable-mdpi/icon_webview.png",
     "java/res_icon/drawable-xhdpi/icon_webview.png",
     "java/res_icon/drawable-xxhdpi/icon_webview.png",
+    "java/res_icon/drawable-xxxhdpi/icon_webview.png",
   ]
 }
 
-- 
2.25.1

